shinydashboard/lantern/modules/locationmodule.R

```diff
@@ -0,0 +1,53 @@
+# Location Module
+library(leaflet)
+
+locationmodule_UI <- function(id) {
+  ns <- NS(id)
+  tagList(
+    h3("Map of Zip Codes with identified organization"),
+    p(strong("Demonstration Only:"), "This map is for demonstration purposes and is still a work in progress."),
```

Comment:
Should this still be here? I know we had it when the map was just an image so I wasn't sure if we should keep it now that we're using leaflet